docs/src/concepts.md

```diff
@@ -0,0 +1,36 @@
+# Concepts
+
+## Modeling the Brain
+
+Studies of brain anatomy and cellular morphology were once confined to single-subject studies wherein in vitro slices were compared to develop a complete model of the brain. With dyes, researchers could compare recordings to the reconstructed location of a given recording device. Through clever experimental design, researchers could further associate highly localized regions to their task-dependent function. Advances in 3D modeling have permitted parallel advances in multi-subject averaged anatomical models. An atlas serves as a shared reference frame for a given species. [The Allen Institute](https://mouse.brain-map.org/) has been a leader in the development of mouse atlases since 2007<sup>[1](#references)</sup>, with regular updates<sup>[2](#references)</sup> that provide researchers with a shared reference frame in the study of functional neuroanatomy.
```

Comment:
1. Please hard wrap to make future edits easier to review
2. How do the footnotes look in local deployment? No Issues? I know mkdocs has their own extension for [footnotes](https://squidfunk.github.io/mkdocs-material/reference/footnotes/#on-multiple-lines), but I wouldn't add it if this gets the job done

Reply:
There are no issues in local deployment with that style for footnotes